### PR TITLE
add recipe for pytest-notification

### PR DESCRIPTION
**Pull request by rhpvorderman**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rhpvorderman** [comment]
I am willing to maintain my own package.

File: recipes/pytest-notification/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+requirements:
+  host:
+    - pip
+    - pytest >=4
+    - python >=3.5
+    - setuptools
+  run:
+    - pytest >=4
+    - python >=3.5
+    - setuptools
```
Comment (marcelotrevisani) [inline recipes/pytest-notification/meta.yaml]: Just to double-check, do you really need ``setuptools`` to run this project?

Comment (rhpvorderman) [reply recipes/pytest-notification/meta.yaml]: @marcelotrevisani Thanks for noticing! This was added by conda skeleton while it is not listed in the requirements. I removed it. Thanks!

**Comment by rhpvorderman** [comment]
@conda-forge/help-python This package is ready for merge.

**Pull request merged**